### PR TITLE
Add support for Inventec d7054q28b.

### PR DESCRIPTION
**Pull request by ghjoec**

### CODE REVIEW

File: packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    int  fd, len;
+    char  buf[10] = {0};
+
+    /* Get each fan status
+     */
+    for (i = 1; i <= NUM_OF_FAN_ON_MAIN_BROAD; i++)
+    {
+        onlp_fan_info_t fan_info;
+
+        if (onlp_fani_info_get(ONLP_FAN_ID_CREATE(i), &fan_info) != ONLP_STATUS_OK) {
```
Comment (jnealtowns) [inline packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c]: This must call onlp_fan_info_get, not onlp_fani_info_get(). 
This function does not run under the API lock.

File: packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+        if (fan_info.status & ONLP_FAN_STATUS_FAILED) {
+            AIM_LOG_ERROR("Fan(%d) is not working, set the other fans as full speed\r\n", i);
+            return onlp_fani_percentage_set(ONLP_FAN_ID_CREATE(1), FAN_DUTY_CYCLE_MAX);
+        }
+
+        /* Decision 1.1: Set fan as full speed if any fan is not present.
+         */
+        if (!(fan_info.status & ONLP_FAN_STATUS_PRESENT)) {
+            AIM_LOG_ERROR("Fan(%d) is not present, set the other fans as full speed\r\n", i);
+            return onlp_fani_percentage_set(ONLP_FAN_ID_CREATE(1), FAN_DUTY_CYCLE_MAX);
```
Comment (jnealtowns) [inline packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c]: Change to onlp_fan_percentage_set()

File: packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+		break;
+	}
+
+	if (i == arr_size) {
+        return onlp_fani_percentage_set(ONLP_FAN_ID_CREATE(1), policy[0].duty_cycle);
+	}
+
+    /* Get current temperature
+     */
+    if (onlp_thermali_info_get(ONLP_THERMAL_ID_CREATE(2), &thermal_1) != ONLP_STATUS_OK ||
```
Comment (jnealtowns) [inline packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c]: All of these must be _theral_info_get

File: packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+	else if ((temp <= policy[i].temp_down_adjust) && (i != 0)) {
+	    new_duty_cycle = policy[i-1].duty_cycle;
+	}
+
+	if (new_duty_cycle == cur_duty_cycle) {
+        /* Duty cycle does not change, just return */
+	    return ONLP_STATUS_OK;
+	}
+
+    return onlp_fani_percentage_set(ONLP_FAN_ID_CREATE(1), new_duty_cycle);
```
Comment (jnealtowns) [inline packages/platforms/inventec/x86-64/x86-64-inventec-d7054q28b/onlp/builds/src/module/src/sysi.c]: _fan_

**Comment by ghjoec** [comment]
Closing this request and issuing new one.

**Pull request closed without merging**